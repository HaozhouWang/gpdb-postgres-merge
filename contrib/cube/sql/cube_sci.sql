---
--- Testing cube output in scientific notation.  This was put into separate
--- test, because has platform-depending output.
---

SELECT '1e27'::cube AS cube;
SELECT '-1e27'::cube AS cube;
SELECT '1.0e27'::cube AS cube;
SELECT '-1.0e27'::cube AS cube;
SELECT '1e+27'::cube AS cube;
SELECT '-1e+27'::cube AS cube;
SELECT '1.0e+27'::cube AS cube;
SELECT '-1.0e+27'::cube AS cube;
SELECT '1e-7'::cube AS cube;
SELECT '-1e-7'::cube AS cube;
SELECT '1.0e-7'::cube AS cube;
SELECT '-1.0e-7'::cube AS cube;
<<<<<<< HEAD
SELECT '1e-700'::cube AS cube;
SELECT '-1e-700'::cube AS cube;
=======
SELECT '1e-300'::cube AS cube;
SELECT '-1e-300'::cube AS cube;
>>>>>>> 9e1c9f959422192bbe1b842a2a1ffaf76b080196
SELECT '1234567890123456'::cube AS cube;
SELECT '+1234567890123456'::cube AS cube;
SELECT '-1234567890123456'::cube AS cube;
