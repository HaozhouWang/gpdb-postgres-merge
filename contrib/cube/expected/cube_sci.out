---
--- Testing cube output in scientific notation.  This was put into separate
--- test, because has platform-depending output.
---
SELECT '1e27'::cube AS cube;
  cube   
---------
 (1e+27)
(1 row)

SELECT '-1e27'::cube AS cube;
   cube   
----------
 (-1e+27)
(1 row)

SELECT '1.0e27'::cube AS cube;
  cube   
---------
 (1e+27)
(1 row)

SELECT '-1.0e27'::cube AS cube;
   cube   
----------
 (-1e+27)
(1 row)

SELECT '1e+27'::cube AS cube;
  cube   
---------
 (1e+27)
(1 row)

SELECT '-1e+27'::cube AS cube;
   cube   
----------
 (-1e+27)
(1 row)

SELECT '1.0e+27'::cube AS cube;
  cube   
---------
 (1e+27)
(1 row)

SELECT '-1.0e+27'::cube AS cube;
   cube   
----------
 (-1e+27)
(1 row)

SELECT '1e-7'::cube AS cube;
  cube   
---------
 (1e-07)
(1 row)

SELECT '-1e-7'::cube AS cube;
   cube   
----------
 (-1e-07)
(1 row)

SELECT '1.0e-7'::cube AS cube;
  cube   
---------
 (1e-07)
(1 row)

SELECT '-1.0e-7'::cube AS cube;
   cube   
----------
 (-1e-07)
(1 row)

<<<<<<< HEAD
SELECT '1e-700'::cube AS cube;
 cube 
------
 (0)
(1 row)

SELECT '-1e-700'::cube AS cube;
 cube 
------
 (0)
(1 row)

SELECT '1234567890123456'::cube AS cube;
          cube          
------------------------
 (1.23456789012346e+15)
(1 row)

SELECT '+1234567890123456'::cube AS cube;
          cube          
------------------------
 (1.23456789012346e+15)
(1 row)

SELECT '-1234567890123456'::cube AS cube;
          cube           
-------------------------
 (-1.23456789012346e+15)
=======
SELECT '1e-300'::cube AS cube;
   cube   
----------
 (1e-300)
(1 row)

SELECT '-1e-300'::cube AS cube;
   cube    
-----------
 (-1e-300)
(1 row)

SELECT '1234567890123456'::cube AS cube;
          cube           
-------------------------
 (1.234567890123456e+15)
(1 row)

SELECT '+1234567890123456'::cube AS cube;
          cube           
-------------------------
 (1.234567890123456e+15)
(1 row)

SELECT '-1234567890123456'::cube AS cube;
           cube           
--------------------------
 (-1.234567890123456e+15)
>>>>>>> 9e1c9f959422192bbe1b842a2a1ffaf76b080196
(1 row)

